src/utils/data-utils/src/schema/format.rs

```diff
@@ -39,6 +39,15 @@ pub fn write_schema_parquet_json(
     Ok(())
 }
 
+pub fn write_schema_arrow_json(
+    output: &mut dyn Write,
+    schema: &Schema,
+) -> Result<(), std::io::Error> {
+    let mut writer = ArrowJsonSchemaWriter::new(output);
```

Comment:
you don't need this custom writer type - simply use `serde_json::to_writer(output, schema)`
https://docs.rs/serde_json/latest/serde_json/fn.to_writer.html

Reply:
hey @sergiimk! thanks for review! done.